tuist/tuist
# Fix ProjectAutomation forceConfigCacheDirectory

Resolves https://github.com/tuist/tuist/issues/4051

### Short description 📝

This PR fixes issues with running plugins via `swift run ...` - this is because we always set `TUIST_CONFIG_FORCE_CONFIG_CACHE_DIRECTORY` env variable even if it's empty - this then leads to failure of constructing `AbsolutePath` from an empty string.

### How to test the changes locally 🧐

Check out this branch and the [swiftlint](https://github.com/tuist/tuist-plugin-swiftlint) plugin, change tuist to be local in the `Package.swift` of the plugin `.package(name: "tuist", path: "path-to-your-tuist"),`

### Checklist ✅
